nucliadb_protos/knowledgebox.proto

```diff
@@ -124,6 +124,7 @@ message EntitiesGroup {
     map<string, Entity> entities = 1;
     string title = 2;
     string color = 3;
+    bool custom = 4;
```

Comment:
According to https://developers.google.com/protocol-buffers/docs/proto#other , bools on proto2 have a default value of `False`, so we should be fine with already existing entities.